packages/protocol/src/data/task-data.ts

```diff
@@ -18,22 +18,27 @@ export interface CustomField {
   value: string;
 }
 
+export namespace TaskDataAccess {
+  export const SINGLE_PREFIX = 'config/task';
+  export const MULTI_PREFIX = 'config/tasks';
+}
+
 export interface TaskDataAccess {
-  'config/task/name': string;
-  'config/task/description': string;
-  'config/task/category': string;
-  'config/task/priority': string;
+  name: string;
+  description: string;
+  category: string;
+  priority: string;
```

Comment:
looks simple :+1: 
... maybe we should/Could already extract the 'expiry' to sub-interface too ... and keep the selector path a single key.

Reply:
currently not possible, needs to be a string with a "/" as a separator. But I'm also not very happy with this...